stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -856,6 +959,24 @@ public func _graph<State : _TensorArrayProtocolEnhanced,
   }
 }
 
+
+public func _tffunc<State : _TensorArrayProtocolEnhanced,
+                    Data : TensorGroup>(
+  with state: State,
+  in fn: @escaping (State, Data) -> (State)
+) -> (Data) -> (String) {
```

Comment:
```suggestion
) -> (Data) -> String {
```

Reply:
Done.

Reply:
Is this done? It seems the text is unchanged `(String)` instead of `String`.